Traceback (most recent call last):
  File "/opt/anaconda3/envs/echem/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/anaconda3/envs/echem/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/echem/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/echem/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/echem/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/opt/anaconda3/envs/echem/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/anaconda3/envs/echem/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import multipsi as mtp

npoints = 21
CIDrv =  mtp.CIDriver()
space = mtp.OrbSpace(mol, scf_drv.mol_orbs)
space.FCI()

energies = []
for i in range(npoints):
    lambda_val = i / (npoints-1)
    CIDrv.UpdateInts(mol.nuclear_repulsion_energy(), H0_1 + lambda_val* V_1, lambda_val* V_2)
    CIDrv.compute(mol, basis, space, 1)
    energies.append(CIDrv.getEnergy(0))
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
Cell [0;32mIn[15], line 6[0m
[1;32m      4[0m CIDrv [38;5;241m=[39m  mtp[38;5;241m.[39mCIDriver()
[1;32m      5[0m space [38;5;241m=[39m mtp[38;5;241m.[39mOrbSpace(mol, scf_drv[38;5;241m.[39mmol_orbs)
[0;32m----> 6[0m [43mspace[49m[38;5;241;43m.[39;49m[43mFCI[49m()
[1;32m      8[0m energies [38;5;241m=[39m []
[1;32m      9[0m [38;5;28;01mfor[39;00m i [38;5;129;01min[39;00m [38;5;28mrange[39m(npoints):

[0;31mAttributeError[0m: 'OrbSpace' object has no attribute 'FCI'

